link/src/main/java/com/stripe/android/link/ui/paymentmethod/PaymentMethodBody.kt

```diff
@@ -124,10 +170,35 @@ internal fun PaymentMethodBody(
             style = MaterialTheme.typography.h2,
             color = MaterialTheme.colors.onPrimary
         )
-        PaymentsThemeForLink {
-            formContent()
+        if (supportedPaymentMethods.size > 1) {
+            Row(
+                Modifier
+                    .fillMaxWidth()
+                    .padding(bottom = 16.dp)
+            ) {
+                supportedPaymentMethods.forEachIndexed { index, paymentMethod ->
+                    PaymentMethodTypeCell(
+                        paymentMethod = paymentMethod,
+                        selected = paymentMethod == selectedPaymentMethod,
+                        enabled = !primaryButtonState.isBlocking,
+                        onSelected = {
+                            onPaymentMethodSelected(paymentMethod)
+                        },
+                        modifier = Modifier.padding(
+                            start = if (index > 0) 10.dp else 0.dp,
+                            end = if (index < supportedPaymentMethods.lastIndex) 10.dp else 0.dp
+                        )
```

Comment:
Instead of this, you can set the `horizontalArrangement` on `Row`:

```kotlin
Row(
  horizontalArrangement = Arrangement.spacedBy(10.dp),
  modifier = …
) {
  …
}
```